components/tyk-dev-portal/values.yaml

```diff
@@ -57,10 +57,11 @@ storage:
   # User can set the storage type for portal.
   # Supported types: fs, s3, db
   type: "fs"
+  dialect: "sqlite3"
```

Comment:
can you please add comments about these fields? since they are not under `database` field, it is confusing to see `type` and `dialect` at the same time. it is not visible to see possible values for `dialect` as well.

Reply:
documentation added :D